website/source/api/secret/kv/versioned-kv.html.md

```diff
@@ -18,13 +18,56 @@ This documentation assumes the kv secrets engine is enabled at the
 possible to enable secrets engines at any location, please update your API calls
 accordingly.
 
+
+## Configure the Versioned K/V Backend
+
+This path configures backend level settings that are applied to every key in the
+key-value store.
+
+| Method   | Path                         | Produces               |
+| :------- | :--------------------------- | :--------------------- |
+| `POST`   | `/secret/config`             | `204 (empty body)`     |
+| `PUT`    | `/secret/config`             | `204 (empty body)`     |
+
+### Parameters
+
+- `max_versions` `(int: 0)` – The number of versions to keep per key. This value
+  applies to all keys, but a key's metadata setting can overwrite this value.
+  Once a key has more than the configured allowed versions the oldest version
+  will be permanently deleted. Defaults to 10. 
+
+- `cas_required` `(bool: false)` – If true all keys will require the cas
```

Comment:
This can be overridden by per-key metadata so I'd put the same info here too.

Reply:
IIRC we decided individual keys couldn't overwrite this value